userguide/storage.rst

```diff
@@ -1295,6 +1295,23 @@ These permissions control the actions users can perform on the dataset
 contents. ACLs are typically used to manage user interactions with
 :ref:`shared datasets <Sharing>`.
 
+The ACL for a new file or directory is typically determined by the
+parent directory ACL. An exception is when there are no *File* or
```

Comment:
I think this should be `*File Inherit*` and the next sentence factored out the repeated word. We should include it for clarity.